contents/docs/self-host/deploy/upgrade-notes.md

```diff
@@ -132,3 +132,25 @@ If you didn’t make any customization to those, there’s nothing you need to d
     - `clickhouse.persistentVolumeClaim` -> `clickhouse.persistence.existingClaim`
 
     If you are overriding any of those, please make the corresponding changes before upgrading. Depending on your settings and setup, during this upgrade the ClickHouse pod might get recreated.
+
+### Upgrading from 10.x.x
+
+11.0.0 removes some legacy Helm annotations not needed anymore. By removing those and upgrading your installation, all future upgrades to stateless components should now happen without downtime (see [#179](https://github.com/PostHog/charts-clickhouse/pull/179) for more info).
```

Comment:
n00b question: is it also no downtime if you have only 1 instance (i.e. upgrade spins up the new one, shifts traffic before nuking the old one)?

Reply:
Yes, now k8s will wait for the new pod to be available before terminating the other